lib/shelley/src/Cardano/Wallet/Shelley.hs

```diff
@@ -449,195 +490,7 @@ exitCodeApiServer = \case
     ListenErrorOperationNotPermitted -> 13
 
 getServerUrl :: Maybe TlsConfiguration -> Socket -> IO URI
-getServerUrl tlsConfig = fmap (fromJust . parseURI . uri) . getSocketName
+getServerUrl tlsConfig = (fromJust . parseURI . uri <$>) . getSocketName
   where
     uri addr = scheme <> "://" <> show addr <> "/"
-    scheme = case tlsConfig of
-                Just _ -> "https"
-                Nothing -> "http"
-
-{-------------------------------------------------------------------------------
-                                    Logging
--------------------------------------------------------------------------------}
```

Comment:
We currently follow the convention that `Logging` and `Errors` are put in the module where the corresponding types / functions are defined. Not sure if moving to an extra module is worth it — the original module is bloated a bit, but there's much less resistance to adding more logging this way. 🤔